datadog_checks_base/datadog_checks/base/stubs/aggregator.py

```diff
@@ -293,6 +293,46 @@ def assert_all_metrics_covered(self):
             missing_metrics = self.not_asserted()
         assert self.metrics_asserted_pct >= 100.0, 'Missing metrics: {}'.format(missing_metrics)
 
+    def assert_no_duplicate_all(self):
```

Comment:
I think we should start putting docstrings in assertions. What is this checking for? what is considered a duplicate and that not?

Reply:
Docs added.